dali/c_api/c_api.cc

```diff
@@ -547,6 +563,23 @@ device_type_t daliGetOutputDevice(daliPipelineHandle *pipe_handle, int id) {
   return device_type_t::CPU;
 }
 
+
+int daliHasOperatorTrace(daliPipelineHandle *pipe_handle, const char *operator_name,
+                         const char *trace_name) {
+  auto *ws = reinterpret_cast<dali::Workspace *>(pipe_handle->ws);
+  auto traces = ws->GetOperatorTraces(operator_name);
+  return traces.find(trace_name) != traces.end() ? 0 : 1;
+}
+
+
+const char *
+daliGetOperatorTrace(daliPipelineHandle *pipe_handle, const char *operator_name,
+                     const char *event_name) {
+  auto *ws = reinterpret_cast<dali::Workspace *>(pipe_handle->ws);
+  return ws->GetOperatorTraces(operator_name).at(event_name).c_str();
```

Comment:
What if there is not `event_name`? Will we get an exception?

Reply:
Yes, the exception would be thrown. Added the exception handling here, since I believe we don't want to throw an exception from the C API.